packages/cli/src/commands/tile/create.ts

```diff
@@ -6,34 +6,25 @@ import { Command } from '../../command'
 import type { CommandFlags } from '../../command'
 
 type Flags = CommandFlags & {
-  'only-genesis'?: boolean
   metadata?: TileMetadataArgs
+  content?: string
 }
 
 export default class CreateTile extends Command<Flags, { content: string }> {
   static description = 'create a new Tile stream'
 
-  static args = [
-    {
-      name: 'content',
-      description: 'stream contents (JSON encoded as string)',
-      required: true,
-      parse: JSON.parse,
-    },
-  ]
-
   static flags = {
     ...Command.flags,
-    'only-genesis': flags.boolean({
-      char: 'g',
-      description: 'only generate genesis commit',
-      default: false,
-    }),
     metadata: flags.string({
       char: 'm',
       description: 'stream metadata',
       parse: JSON.parse,
     }),
+    content: flags.string({
+      char: 'b',
```

Comment:
why 'b'?  why not 'c'?

Reply:
ah.  and I guess using `-u` for "url" could be confused with "user".  Oh well, not a big deal

Reply:
Maybe we can remove the `char` to prevent confusion here? It shouldn't be a big deal to have to set `--content` rather than `-b` when running this command.

Reply:
I'm a little opposed to removing the `char` overall. It's not a big deal at the moment because I don't think the CLI sees a lot of use, but overall I can see only allowing `--content` becoming a sticking point with users. maybe `-j` for JSON?

Reply:
it might make sense to update the `-c` from the parent command to being `--ceramic` only instead. It'd be a bit more explicit and also encourage users to work with the local files for more commands.

Reply:
Could we remove it from now and revisit later as part of a larger spec for the CLI please?
I'd rather not change the `--ceramic` / `-c` flag now as there are likely docs or blog posts using it, and I'd rather not introduce a `-b` flag that doesn't have direct meaning and might be removed later.

Reply:
Totally ok! will implement this today & close :)

Reply:
Thanks!